.github/workflows/build.yml

```diff
@@ -35,9 +39,8 @@ jobs:
         run:  mvn -B package
 
       - name: Conditionally Deploy to Igniterealtime Archiva
-        if: ${{github.repository == 'igniterealtime/openfire-sip-plugin' && github.ref == 'refs/heads/master' && github.event_name == 'push' && matrix.java == '1.8'}}
+        if: ${{ contains(github.repository, 'igniterealtime/') && ( github.ref == 'refs/heads/master' || contains(github.ref, 'refs/tags/') ) && github.event_name == 'push' && matrix.java == '1.8'}}
```

Comment:
This will only run for pushes, not for "create" of a tag. Would "release" be worth considering? Is that simpler, or just a less specific abstraction?

Reply:
These are good questions, my preference is to go off the tags as that's more atomic to what we want to build releases from.  One could make tags without a release, for example.  

Oye, I see what you are saying now about pushes, sigh.